scripts/drtprod

```diff
@@ -43,10 +43,23 @@ case $1 in
     ;;
   "create")
     if [ "$#" -lt 2 ]; then
-      echo "usage: $0 create {drt-ua1,drt-ua2}"
+      echo "usage: $0 create {drt-main,drt-ua1,drt-ua2}"
       exit 1
     fi
     case "${2}" in
+      "drt-main")
+        roachprod create drt-main \
+          --clouds gce \
+          --gce-managed \
+          --gce-zones "us-east1-b,us-west1-b,europe-west2-b" \
+          --nodes 15 \
+          --gce-machine-type n2-standard-16 \
+          --gce-pd-volume-size 10000 --local-ssd=false \
+          --username drt \
+          --lifetime 8760h
+        # setup dns
+        $0 dns drt-main
```

Comment:
this will fail on initial creation apparently update is distinct from create

fix is out here: https://github.com/cockroachdb/cockroach/pull/120463

Reply:
great, thanks I'll rebase once https://github.com/cockroachdb/cockroach/pull/120463 is merged

Reply:
IIRC git will just elide the repeated commit in the merge if you bors without waiting